src/components/DisplayEmbed/DisplayExternalVisualElement.jsx

```diff
@@ -43,12 +44,13 @@ export class DisplayExternalVisualElement extends Component {
 
   async getPropsFromEmbed() {
     const { embed } = this.props;
-    const domain = urlDomain(embed.url);
+    const domain = config.h5pApiUrl;
```

Comment:
Vil ikkje dette kunne knekke visuelle element som er fra brightcove?

Reply:
Skal egentlig ikke det, for fila som kaller på `DisplayExternalVisualElement` er `DisplayVisualElement` og der gjøres sjekk på embed type. Brightcove har sin egen komponent. 

```
switch (embed.resource) {
    case 'image': (...)
    case 'brightcove': (...)
    case 'external':
    case 'iframe':
    case 'h5p':
      return (
        <DisplayExternalVisualElement
          embed={embed}
          changeVisualElement={changeVisualElement}
          onRemoveClick={onRemoveClick}
          onFigureInputChange={e =>
            onChange({
              target: { name: 'visualElementCaption', value: e.target.value },
            })
          }
          language={language}
        />
      );
```

Før var det bare `case 'iframe'` og `case 'h5p'` som kallte på `DisplayExternalVisualElement` og jeg la til `case 'h5p'`

Reply:
Men kan hende at det kan påvirke `external` typene, skal se nærmere på dette

Reply:
Visual element kan være eksternt element via iframe-komponent så det kan tenkes at det har noko å sei.

Reply:
Forsøkte å sette inn https://vimeo.com/146755411 som visuelt element i PR og det funka ikkje. På ed.test fungerer det.